tests/Unit/Helpers/DataStructures/Tensor/Expressions/EvaluateRank3TestHelpers.hpp

```diff
@@ -11,8 +11,12 @@
 #include "DataStructures/Tensor/Expressions/TensorExpression.hpp"
 #include "DataStructures/Tensor/Tensor.hpp"
 #include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/Literals.hpp"
 #include "Utilities/TMPL.hpp"
 
+namespace TestHelpers {
```

Comment:
clang-tidy will want you to do `TestHelpers::TensorExpressions`. This type of namespace scoping is a C++17 feature :)